hw1/02-palindrome.js

```diff
@@ -1,3 +1,36 @@
-const elem = document.querySelector('input');
-
-elem.addEventListener('input', handleInput);
+document.addEventListener("DOMContentLoaded", () => {
+    const output = document.getElementById("output");
+    const numberInput = document.getElementById("numberInput");
+  
+    numberInput.addEventListener("input", function() {
+      const value = this.value;
+  
+      // If empty
+      if (value === "") {
+        output.innerHTML = "";
+        return;
+      }
+  
+      // If value is less than 0 OR the value was a non-number (i.e.: text, symbols, etc.)
+      if (value < 0 || isNaN(value)) {
+        output.innerHTML = '<span class="text-danger">Please enter a valid positive number.</span>';
+        return;
+      }
```

Comment:
I am struggling to trigger this for conditional for symbols. Instead the output remains blank.